### ISSUE TITLE
Questions about computation of information and correlation dimensions

### ISSUE THREAD

**Issue by ignaciorlando** [m-reuter/fdim#1]
I'm using as a reference your thesis in http://reuter.mit.edu/blue/papers/reuter-fractal/reuter-fractal.pdf. However, I observed that your equations for computing information and correlation dimensions are significantly different that the functions that you implemented. Would it be possible for you to provide me with references indicating that your implementation is an alternative way to compute the very same fractal dimensions that you indicated in your thesis? Thank you so much!

**Comment by m-reuter** [comment]
Strange, the equations and implementation should be the same.

**Comment by ignaciorlando** [comment]
Hello @m-reuter, thanks for your fast response. I'm sorry, maybe I wasn't clear enough. I'm using your Matlab implementation (the matlab/getFD function specifically). 

If I understand correctly, the variables L, NL, InfL and SqrFreqL should correspond with \lambda (scales of the boxes), N(\lambda) (first equation in page 9), I(\lambda) (second equation in page 13) and C(\lambda) (second/third equations in page 14) in your thesis. 

However, the process to calculate each of them in lines 59-60 in matlab/getFD is not the same as the one explained in your thesis. Am I being right? Is it possible to ensure that the process you follow in lines 59-60 is mathematically equivalent to the equations in your thesis? Moreover, do you have any english references on how this fractal dimensions can be obtained? 

I would really appreciate it as I would like to complement the experiments in a paper in which I'm working on with grayscale fractal dimensions.

Kind regards,

@ignaciorlando

**Comment by m-reuter** [comment]
I am sorry, but I just moved between countries. I cannot look into the code right now. The matlab stuff has been coded last year, so it may differ from the C implementation, but I checked for differences and don’t remember finding any during development. Compare the results you are getting from the matlab implementation with the ones from the fdim GUI.
The C++ code was done during the thesis project. So I think both should match the math. Take a look at all references that I list in the thesis and probably elsewhere (maybe in the source code).
